server/routes/api/campaigns.js

```diff
@@ -25,4 +26,16 @@ router.get('/', async (req, res) => {
   }
 })
 
+// Delete endpoint
```

Comment:
This route looks pretty good, but it should be in the v1 campaign router instance 🙂

Reply:
The delete route has been transferred to the v1/campaigns folder